src/pages/root/Root.js

```diff
@@ -0,0 +1,28 @@
+import { GlobalStyle } from 'assets/styles/GlobalStyle.js';
+import { ThemeProvider } from 'styled-components';
+import { theme } from 'assets/styles/theme.js';
+import { Wrapper } from 'assets/styles/Root.styles';
+import { Header } from 'components/common/Header.js';
+import 'assets/styles/Root.styles';
+import { List } from 'pages/mainView/List';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { MoreInfo } from 'pages/moreInfo/MoreInfo';
+
+const Root = () => {
+  return (
+    <Router>
+      <ThemeProvider theme={theme}>
+        <GlobalStyle />
+        <Wrapper>
+          <Header />
+          <Routes>
+            <Route path="/more-info" element={<MoreInfo />} />
```

Comment:
Jedyna rzecz do której mogę się przyczepić w tym PR, to brak parametrów przy nawigowaniu do podstrony "more-info". Najczęsciej w produkcyjnych aplikacjach, gdy masz podstronę ze szczegółami elementu z listy, to w linku znajduje się odpowiednie ID. W ten sposób, gdy udostępnisz komus ten link, to będzie mógł go "wkleić" w przeglądarkę i wyswietlić odpowiednią stronę. Dana strona dzięki id w parametrach linku wie dokładnie jakie dane pobrać i wyswietla odpowiednią podstronę. W momencie, gdy nawiguje się wyłącznie do /more-info to nie ma żadnych parametrów (pojawia się error). 

https://reactrouter.com/docs/en/v6/getting-started/overview na overview react router są przykłady jak przekazywać id w linku. 

Najlepiej zadeklarować jako <Route path="more_info/:trackId" element={<MoreInfo />} />, a z ListItem nawigować przekazując id: navigate(`/more-info/${trackId}`).